data/resources/ui/content-chat-info-member-row.ui

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="ContentChatInfoMemberRow" parent="GtkBox">
```

Comment:
Subclassing from GtkBox is bad practice. You expose all the unneeded methods and functionality, which we don't want. For custom widgets you usually have to just subclass from `GtkWidget`  and then set a layout manager (`GtkBoxLayout` in this case). [This is an example of how it can be done](https://github.com/melix99/telegrand/blob/633a93f011413ad24235ca16f3c7aeb687ab1f24/src/session/sidebar/search/item_row.rs#L54).